projects/plugins/jetpack/extensions/blocks/google-docs-embed/edit.js

```diff
@@ -116,7 +116,7 @@ const GsuiteBlockEdit = props => {
 		 * If the block is not the expected one, return the
 		 * original URL as is.
 		 */
-		if ( [] === patterns[ 0 ] || '' === type ) {
+		if ( patterns.length === 0 || '' === type ) {
```

Comment:
Best I could figure out for what was intended here.

Reply:
Yeah, it's definitely weird: patterns should be an array of regexes, not a multidimensional array.